src/Services/ConvertToLearnosityService.php

```diff
@@ -364,7 +414,7 @@ private function convertAssessmentItem($xmlString, $itemReference = null, $resou
         AssumptionHandler::flush();
 
         $xmlString = CheckValidQti::preProcessing($xmlString);
-
+        
```

Comment:
Formatting: Trailing whitespace is appearing again. You might want to check your editor settings and see if you can get it to remove trailing whitespace on save (or you can manually do so yourself).

Reply:
Done